packages/vite-extensions/vite-website-cache-paths.ts

```diff
@@ -0,0 +1,93 @@
+import { readdirSync, statSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+/**
+ * We want to cache all files from the website root directory, and the assets directory, except for the files listed below.
+ */
+const patternsToNotCache = [
+	/**
+	 * Static files that are not needed for the website to function offline.
+	 * TODO: Check if these are needed in the production build.
```

Comment:
Is this TODO up to date? It feels like this PR hinges on that since the offline mode only works in production build. FWIW I think the list is legit and we're okay to exclude these files.

Reply:
```suggestion
```

Reply:
This comment should have been an issue, so I opened one now https://github.com/WordPress/wordpress-playground/issues/1612